.github/workflows/release.yml

```diff
@@ -1,5 +1,8 @@
 name: Release
 
+permissions:
+  id-token: write
+
```

Comment:
Can you move this to the `release` job? Since the `check-package-version` job doesn't need this permission

Reply:
Oh and I believe you'll also need `contents: read` to be able to check out the repo